build-elements/ff4j-features.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs-extra');
+const concat = require('concat');
+
+(async function build() {
+  const files = [
+    './dist/ff4j-features/runtime-es5.js',
+    './dist/ff4j-features/polyfills-es5.js',
+    './dist/ff4j-features/main-es5.js'
+  ];
+
+  await fs.ensureDir('elements');
+  await concat(files, 'elements/ff4j-features.js');
+  await fs.copyFile(
+    './dist/ff4j-features/styles.css',
+    'elements/ff4j-features.styles.css'
+  );
+}());
```

Comment:
Refer https://github.com/paul58914080/micro-fontend-todo-kata-final/blob/master/projects/create-todo/build/bundle.js#L1-L14

```javascript
const fs = require('fs-extra');
const concat = require('concat');
const del = require('del');

(async function build() {
  const files = [
    'dist/create-todo/runtime-es2015.js',
    'dist/create-todo/polyfills-es2015.js',
    'dist/create-todo/main-es2015.js'
  ];
  await del(['dist/elements/create-todo.js']);
  await fs.ensureDir('dist/elements');
  await concat(files, 'dist/elements/create-todo.js');
})();
```

You would have to delete and then concatenate. Style is not required, I think is is built in

Reply:
Yeah, I was aware but is it mandatory? because even without that, it ll first delete and replace I think